workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/WorkerBuildData.cs

```diff
@@ -19,7 +21,35 @@ public class WorkerBuildData
 
         private readonly BuildTarget buildTarget;
 
-        private static readonly Dictionary<BuildTarget, string> BuildTargetNames =
+        internal static readonly IReadOnlyList<BuildTarget> AllBuildTargets = new List<BuildTarget>
+        {
+            BuildTarget.StandaloneWindows,
+            BuildTarget.StandaloneWindows64,
+            BuildTarget.StandaloneLinux64,
+            BuildTarget.StandaloneOSX,
+            BuildTarget.Android,
+            BuildTarget.iOS,
+        };
+
+        internal static readonly IReadOnlyList<BuildTarget> LocalBuildTargets = new List<BuildTarget>
+        {
+            CurrentBuildPlatform,
```

Comment:
@zeroZshadow found that this sets the local platform on a mac to be `Win x64`

this leads to mac TC agents attempting to build il2cpp UnityClients for windows, when such a package doesn't exist on mac

TC should work fine on mac again once this is fixed :)

Reply:
Fixed and re-running :)